.devcontainer/features/azure-functions-core-tools/install.sh

```diff
@@ -0,0 +1,28 @@
+echo "Choose the version of Azure Functions Core Tools to install"
+if [ "$VERSION" = "latest" ]; then
```

Comment:
thought: what about defaulting to latest when undefined?

Reply:
`"latest"` is the default value, so it will be picked as fallback when `"version"` is not specified https://github.com/pagopa/dx/blob/5892d51b560f1b9d61b986a1f6826f4574fc087b/.devcontainer/features/azure-functions-core-tools/devcontainer-feature.json#L16